jack.py

```diff
@@ -2027,6 +2035,15 @@ def register(self, shortname, is_terminal=False, is_physical=False):
         self._portlist.append(port)
         return port
 
+    def unregister_range(self, start=None, end=None, step=1):
```

Comment:
If you want to add some feature to this file, you have to make a PR on spatialaudio/jackclient-python. But with example I send you, you don't need this 2 methods.